Makefile

```diff
@@ -136,6 +136,8 @@ DEFMEMSLOTS := 10
 DEFBRIDGES := 1
 #Default network model
 DEFNETWORKMODEL := macvtap
+#Default entropy source
+DEFENTROPYSOURCE := /dev/random
```

Comment:
Oh sorry, I'd like `urandom` to be the default here, and we should add some comments related to `/dev/random`, explaining this can be used but with some limitations.